client/stylesheets/_modal.scss

```diff
@@ -260,30 +260,30 @@
 }
 
 /* toggle hover */
-.modOfferRequestPaymentWrap ul li input.offerRequestPaymentRadio:hover:not(:checked) ~ label:before {
+.radioWrap ul li input.radioWrapElement:hover:not(:checked) ~ label:before {
 	content:'\2714';
 	text-indent: .9em;
 	color: #C2C2C2;
 }
 
-.modOfferRequestPaymentWrap ul li input.offerRequestPaymentRadio:hover:not(:checked) ~ label {
+.radioWrap ul li input.radioWrapElement:hover:not(:checked) ~ label {
 	color: #888;
 }
 
 /* toggle on */
-.modOfferRequestPaymentWrap ul li input.offerRequestPaymentRadio:checked ~ label:before {
+.radioWrap ul li input.radioWrapElement:checked ~ label:before {
```

Comment:
Merge rule `.radioWrap ul li input.radioWrapElement:checked ~ label:before` with rule on line 215<br>Begin pseudo elements with double colons: `::`<br>Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 4<br>Selector `radioWrap` should be written in lowercase with hyphens<br>Selector `radioWrapElement` should be written in lowercase with hyphens